mle/model.py

```diff
@@ -3,110 +3,72 @@
 import json
 import importlib.util
 from abc import ABC, abstractmethod
-
-from mle.function import get_function, process_function_name
+from typing import List, Dict, Any, Optional
+from mle.utils import process_function_name  # Adjust the import path as needed
```

Comment:
you may remove the comments here :)

Reply:
`from mle.utils import process_function_name` -> `from mle.function import process_function_name`